go.mod

```diff
@@ -8,26 +8,20 @@ require (
 	github.com/sirupsen/logrus v1.4.1
 	github.com/spf13/cobra v0.0.0-20160830174925-9c28e4bbd74e
 	github.com/spf13/viper v0.0.0-20150621231900-db7ff930a189
-	github.com/stellar/go v0.0.0-20221028081946-9308f9531041
```

Comment:
just for suggestion, a comment about the version string, i.e. which part is git hash, maybe a brief how-to/example for changing to a newer version of go repo?

Reply:
I don't know if I can add that comment in here ( i.e. inside the go.mod file ), but I can add a comment in the README file. Beside that, I'm hoping that the deptool would be able to help us with that ( once it's ready for prime time ).
When I did the update, I used
go get github.com/stellar/go@9308f9531041

but I clearly realize it's not the best approach and we need better tooling here.